app/controllers/v0/chatbot/report_to_cxi.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'net/http'
+require 'openssl'
+require 'json'
+
+module V0
+  module Chatbot
+    class ReportToCxi
```

Comment:
Since you're refactoring, can you move this class into `lib/chatbot/report_to_cxi`? I know it was in the controllers folder before but it really should live somewhere else